src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/ParalogyFmsDTOValidator.java

```diff
@@ -20,57 +33,165 @@
+			predictionMethodsMatched = new ArrayList<>();
+			for (String methodName : dto.getPredictionMethodsMatched()) {
+				VocabularyTerm method = vocabularyTermService.getTermInVocabulary(VocabularyConstants.PARALOGY_PREDICTION_METHOD_VOCABULARY, methodName).getEntity();
+				if (method == null) {
+					paralogyResponse.addErrorMessage("predictionMethodsMatched", ValidationConstants.INVALID_MESSAGE + " (" + methodName + ")");
+				} else {
+					predictionMethodsMatched.add(method);
+				}
+			}
+		}
+		paralogyData.setPredictionMethodsMatched(predictionMethodsMatched);
+
+		List<VocabularyTerm> predictionMethodsNotMatched = null;
+		if (CollectionUtils.isNotEmpty(dto.getPredictionMethodsNotMatched())) {
+			predictionMethodsNotMatched = new ArrayList<>();
+			for (String methodName : dto.getPredictionMethodsNotMatched()) {
+				VocabularyTerm method = vocabularyTermService.getTermInVocabulary(VocabularyConstants.PARALOGY_PREDICTION_METHOD_VOCABULARY, methodName).getEntity();
+				if (method == null) {
+					paralogyResponse.addErrorMessage("predictionMethodsNotMatched", ValidationConstants.INVALID_MESSAGE + " (" + methodName + ")");
+				} else {
+					predictionMethodsNotMatched.add(method);
+				}
+			}
+		}
+		paralogyData.setPredictionMethodsNotMatched(predictionMethodsNotMatched);
+
+		List<VocabularyTerm> predictionMethodsNotCalled = null;
+		if (CollectionUtils.isNotEmpty(dto.getPredictionMethodsNotCalled())) {
+			predictionMethodsNotCalled = new ArrayList<>();
+			for (String methodName : dto.getPredictionMethodsNotCalled()) {
+				VocabularyTerm method = vocabularyTermService.getTermInVocabulary(VocabularyConstants.PARALOGY_PREDICTION_METHOD_VOCABULARY, methodName).getEntity();
+				if (method == null) {
+					paralogyResponse.addErrorMessage("predictionMethodsNotCalled", ValidationConstants.INVALID_MESSAGE + " (" + methodName + ")");
+				} else {
+					predictionMethodsNotCalled.add(method);
+				}
+			}
+		}
+		paralogyData.setPredictionMethodsNotCalled(predictionMethodsNotCalled);
+
+		VocabularyTerm confidence = null;
+		if (StringUtils.isBlank(dto.getConfidence())) {
+			paralogyResponse.addErrorMessage("confidence", ValidationConstants.REQUIRED_MESSAGE);
+		} else {
+			confidence = vocabularyTermService.getTermInVocabulary(VocabularyConstants.ORTHOLOGY_CONFIDENCE_VOCABULARY, dto.getConfidence()).getEntity();
+			if (confidence == null) {
+				paralogyResponse.addErrorMessage("confidence", ValidationConstants.INVALID_MESSAGE + " (" + dto.getConfidence() + ")");
+			}
+		}
+		paralogyData.setConfidence(confidence);
+
+		if(dto.getSimilarity() == null){
+			paralogyResponse.addErrorMessage("Similarity", ValidationConstants.REQUIRED_MESSAGE);
+		} else {
+			paralogyData.setSimilarity(dto.getSimilarity());
+		}
+		if(dto.getIdentity() == null){
+			paralogyResponse.addErrorMessage("Identity", ValidationConstants.REQUIRED_MESSAGE);
+		} else {
+			paralogyData.setIdentity(dto.getIdentity());
+		}
+		if(dto.getLength() == null){
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' is not preceded with whitespace.